dashboard/db/migrate/20160426210000_add_levels_to_script_level.rb

```diff
@@ -0,0 +1,33 @@
+class AddLevelsToScriptLevel < ActiveRecord::Migration
+  def up
+    create_join_table :levels, :script_levels do |t|
+      t.index :level_id
+      t.index :script_level_id
+    end
+
+    ScriptLevel.class_eval do
+      belongs_to :old_level, class_name: "Level", foreign_key: "level_id"
+    end
+
+    ScriptLevel.all.each do |scriptlevel|
+      unless scriptlevel.old_level.nil?
+        scriptlevel.levels << scriptlevel.old_level
+        scriptlevel.save
+      end
+    end
+  end
+
+  def down
+    ScriptLevel.class_eval do
+      belongs_to :new_level, class_name: "Level", foreign_key: "level_id"
+    end
+
+    ScriptLevel.all.each do |scriptlevel|
```

Comment:
Not: usual style is to underscorize the same way rails does (so ScriptLevel -> script_level